native/python/pyjp_module.cpp

```diff
@@ -560,7 +566,7 @@ static PyObject* PyJPModule_isPackage(PyObject *module, PyObject *pkg)
 }
 
 
-#if 0
+#if 1
```

Comment:
Do you expect to use this debug code in the future? If not, I recommend getting rid of it. It is stored in the history of the file anyways.

Reply:
This was a leak for another PR.  This particular test code gets used whenever I debug memory issues with slots.  It was reactivated for #933.  Maybe i should just tag it as a branch as it is likely to get reused.  Or perhaps on the instrumentation compile flag.